ee/src/main/java/org/jboss/as/ee/subsystem/ContextServiceResourceDefinition.java

```diff
@@ -38,6 +40,12 @@
  */
 public class ContextServiceResourceDefinition extends SimpleResourceDefinition {
 
+    /**
+     * the resource definition's dynamic runtime capability
+     */
+    public static final RuntimeCapability<Void> CAPABILITY = RuntimeCapability.Builder.of("org.wildfly.ee.concurrent.context.service", true, ContextServiceImpl.class)
```

Comment:
Can we not use javax.enterprise.concurrent.ContextService for the service type?  Generally, we should avoid exposing specific implementation classes via the capability contract.

Reply:
that would introduce loads of changes, I would prefer to stick with same service type.

Reply:
OK - this is fine then, given that these are private capabilities.